giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/service/delete/IDeleteDocumentService.java

```diff
@@ -0,0 +1,19 @@
+package edu.asu.diging.gilesecosystem.web.core.service.delete;
+
+import edu.asu.diging.gilesecosystem.web.core.model.IDocument;
+
+public interface IDeleteDocumentService {
+    /**
+     * Sends storage deletion messages for all the files of the document to nepomuk via kafka
+     * @param document
+     *         The document to be deleted.
+     */
+    void deleteDocument(IDocument document);
```

Comment:
it's not actually deleting anything, right? so maybe `initiateDeletion` or something like that?